core/src/main/java/com/linecorp/armeria/client/Http1ClientKeepAliveHandler.java

```diff
@@ -31,7 +33,7 @@
 import io.netty.channel.ChannelFuture;
 import io.netty.channel.ChannelHandlerContext;
 
-final class Http1ClientKeepAliveHandler extends Http1KeepAliveHandler {
+final class Http1ClientKeepAliveHandler<T extends ConnectionEventKey> extends Http1KeepAliveHandler {
```

Comment:
It looks like the parameter type isn't used.
```suggestion
final class Http1ClientKeepAliveHandler extends Http1KeepAliveHandler {
```